src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+                parsedDescription = parsedDescription.substring(parsedDescription.indexOf(" ") + 1);
+                System.out.println(BARRIER + "\n");
+                Event temp = new Event(parsedDescription, start, end);
+                System.out.println("added: " + temp.printTask());
+                printListLength();
+                System.out.println(BARRIER + "\n");
+                break;
+            }
+
+            case "delete": {
+                try {
+                    if (!isValidInput(splitInput[1])) {
+                            throw new DukeExceptions.invalidNumberException(BARRIER + "\n\nBlast! That isn't a valid number lad!\n" + BARRIER + "\n");
+                    }
+                    int index = Integer.parseInt(splitInput[1]) - 1;
+                    System.out.println(BARRIER + "\n\nAye! The task be removed:");
+                    System.out.println("    " + Task.getTasksArray().get(index).printTask());
+                    Task.getTasksArray().remove(index);
+                    printListLength();
+                    System.out.println(BARRIER + "\n");
+                } finally {
+                    break;
+                }
+            }
+
+            // Adds a new task if no case is hit
+            default: {
+                System.out.println(BARRIER + "\n");
+                System.out.println("Argh! This is not a valid order, type 'help' for assistance!");
+                System.out.println(BARRIER + "\n");
+            }
+            // End of switch
+            }
+
+            // Reads next input
+            input = CONSOLE.nextLine();
+        }
+
+        System.out.println(EXIT);
+
+        // Save files to data.txt
+        try {
+            FileManager.populateFile();
+        } catch (IOException e) {
+            System.out.println("Argh! There was an issue saving yer tasks!");
+        }
     }
-}
+
+    // Local Duke method to list all functions
+    public static void printHelpList() {
+        String BARRIER = "____________________________________________________________";
+        System.out.println(BARRIER + "\n\nAvast! Here be the commands ye can use to make me do yer bidding!" + 
+                                    "\n- list: lists all current tasks\n- mark x: marks task x as complete\n-" + 
+                                    "unmark x: unmarks task x as complete\n- todo 'description': adds a task to do " + 
+                                    "with the given description\n- deadline 'description' /by 'deadline': adds a deadline" + 
+                                    " task with the given date and description\n- event 'description' /from 'start' /to 'end': " +
+                                    "adds an event with the start and endtime\n- bye: exits Duke\n- anything else: " + 
+                                    "adds a basic task with the given description\n" + BARRIER + "\n");
```

Comment:
Consider keeping your indentation for wrapped lines to be 8 spaces from the parent line. This would also help in ensuring that your individual lines are not too long (currently, the longest line is 129 characters, while our coding standard advises the limit to be 120)